PythonTasks/CorSen_ex.py

```diff
@@ -0,0 +1,23 @@
+def correct_sentence(text: str) -> str:
```

Comment:
Are you sure function annotations are needed there? As for me, I haven't met usages of it. I don't think it's pythonic same as `;`.  You can finish your lines with `;` but there is no need in it